plugins/bufferlines/bufferline.nix

```diff
@@ -104,26 +104,25 @@ in
 
       themable = helpers.defaultNullOpts.mkBool true "Whether or not bufferline highlights can be overridden externally";
 
-      numbers =
-        helpers.defaultNullOpts.mkNullable
-          (
-            with types;
-            either (enum [
-              "none"
-              "ordinal"
-              "buffer_id"
-              "both"
-            ]) helpers.nixvimTypes.rawLua
-          )
+      numbers = helpers.defaultNullOpts.mkEnum' rec {
+        values = [
           "none"
-          ''
-            Customize the styling of numbers.
-
-            Either one of "none" "ordinal" "buffer_id" "both" or a lua function:
-            ```
+          "ordinal"
+          "buffer_id"
+          "both"
+        ];
+        default = "none";
+        description = ''
+          Customize the styling of numbers.
+
+          Either one of ${concatStringsSep " " (map (v: ''`"${v}"`'') values)} or a lua function.
```

Comment:
Could probably drop this line, the example should make things clear:
![image](https://github.com/nix-community/nixvim/assets/5046562/6acd7d31-8713-4b87-9b7e-01811a5d2957)